tests/security/test_kerberos.py

```diff
@@ -104,3 +105,255 @@ def test_args_from_cli(self):
                 )
 
             assert ctx.value.code == 1
+
+
+class TestKerberosUnit(unittest.TestCase):
```

Comment:
Should we use a pytest setup instead? `pytest.mark.paramtrize`, `caplog`, and `pytest.raises` are easier to use than `parameterized`, `assertLogs`, and `assertRaises` IMO. But this is minor and I’m OK if you feel the current approach is more familiar etc.

Reply:
This is very subjective, but I checked the popularity of each approach and I can see that it is very close.

PS. We had discussions on the mailing list about this. https://lists.apache.org/thread.html/1e4df7d4b0cd9b2d2bb76a3336471aa85e852545dd41ada6d4e461b8%40%3Cdev.airflow.apache.org%3E Feel free to join it